### PR TITLE
Mascots: rework

### PR DESCRIPTION
**Pull request by MegaApplePi**
Also fixed a broken image link in `Game_Modes/osu!catch` because Mascots/Gallery had also used it too.

~~The fanart selection for Mocha was based on first place drawings from the polls while Maria was based on whichever one I can find first... there isn't much of her in the fanart contests. :frowning:~~

### CODE REVIEW

File: wiki/Mascots/en.md [hunk 1]
```diff
@@ -1,234 +1,99 @@
...
-?
+![Don](img/Don.png "Don")
 
-#### Appearance
-Mania
+> Don is the protagonist of the Taiko no Tatsujin series. He is a taiko drum with a white edge and four legs, a red face (which appears to be moly orange), and a light blue body. Don's dream is to share the beauty of Taiko with the world. Three years have passed since he moved into the Wada House, and he's become quite popular around town. He has a monstrous appetite and sometimes has expensive shopping at the Wada House which may take over for the worst. He usually ends his sentences with "Ta-don", meaning "Ba-dum" in Japanese.
 
-#### Creator
-* Daru (Art)
+和田どん (Wada Don), also known as Don or Don-chan, was one of the osu! mascots for osu!taiko tha joined in 2008-05. He is 48cm tall and weighs 100+ kilograms. He appears in the skin for osu!taiko. His art was designed by Yukiko Yokoo (横尾有希子) and was voiced by Narahashi Miki (楢橋 美紀).
```
Comment (Joehuu) [inline wiki/Mascots/en.md]: tha -> that

File: wiki/Mascots/en.md [hunk 2]
```diff
@@ -1,234 +1,99 @@
...
+### pippi
 
-#### Alisaes
-* Pippi
-* Pippidon
+![pippi](img/pippi.png "pippi")
 
-#### Birthday
-?
+pippi, stylized with a lowercase p, is the osu!standard mascot that joined on 2008-06-22. She is also known as pippidon in osu!taiko and had an appeareance in [Yandere Simulator](https://yanderesimulator.com) as an NPC. The initial concept art was created by Sarumaru, the pippidon sprite were created by crystalsuicune, and the current art was designed by Daru.
```
Comment (Joehuu) [inline wiki/Mascots/en.md]: Idk but in the video showcase, it said pippi's joined date was July 2008.

Comment (Joehuu) [inline wiki/Mascots/en.md]: I think you should link users whenever mentioned.

File: wiki/Mascots/Gallery/en.md [hunk 3]
```diff
@@ -1,46 +1,113 @@
...
+- [Fan Art 8: Christmas](/community/contests/36)
+- [Mocha in Summer Fanart Contest](/community/contests/48)
 
 ### Yuzu
 
-![Yuzu by Daru](img/Yuzu_Artwork.png "Yuzu by Daru")
-![Yuzu in osu!catch by ztrot](img/Yuzu_Idle.png "Yuzu in osu!catch by ztrot")
-![Yuzu during a Kiai Time in osu!catch](img/Yuzu_Kiai.png "Yuzu during a Kiai Time in osu!catch")
-![Yuzu while missing fruits in osu!catch](img/Yuzu_Fail.png "Yuzu while missing fruits in osu!catch")
+![Yuzu by Daru](img/Yuzu-Daru.png "Comboburst  by Daru")
```
Comment (TPGPL) [inline wiki/Mascots/Gallery/en.md]: double space? 👀

File: wiki/Mascots/en.md [hunk 4]
```diff
@@ -1,234 +1,99 @@
...
+### pippi
 
-#### Alisaes
-* Pippi
-* Pippidon
+![pippi](img/pippi.png "pippi")
 
-#### Birthday
-?
+pippi, stylized with a lowercase p, is the osu!standard mascot that joined on 2008-07. She is also known as pippidon in osu!taiko and had an appeareance in [Yandere Simulator](https://yanderesimulator.com) as an NPC. The initial concept art was created by Sarumaru, the pippidon sprite were created by [crystalsuicune](/users/9974), and the current art was designed by [Daru](/users/32480).
```
Comment (TPGPL) [inline wiki/Mascots/en.md]: with a lowercase `P`
had an appearance -> appeared
sprite was/sprites were
link to Sarumaru's profile?

File: wiki/Mascots/en.md [hunk 5]
```diff
@@ -1,234 +1,99 @@
...
 
-#### Height
-?
+### Yuzu
 
-#### Appearance
-* osu! (Official mascot)
-* Taiko (as Pippidon)
-* Yandere Simulator (as NPC easter egg)
+<!-- *For the news post, see: [Meet Yuzu](/home/news/89483664163).* -->
```
Comment (TPGPL) [inline wiki/Mascots/en.md]: ? :eyes:

File: wiki/Mascots/en.md [hunk 6]
```diff
@@ -1,234 +1,99 @@
...
 
-#### Creator
-* Sarumaru (Concept and original art)
-* Daru (Current art)
-* crystalsuicune (pippidon)
+![Yuzu](img/Yuzu.png "Yuzu")
 
-#### Gender
-Female
+Yuzu is the osu!catch mascot that had joined on 2014-06-22. He was born on 2000-04-10, is 172cm tall, and weighs 65kg. His initial art design and catcher sprite was done by [ztrot](/users/6347) while Daru had created the comboburst art.
```
Comment (TPGPL) [inline wiki/Mascots/en.md]: that joined
has created

File: wiki/Mascots/en.md [hunk 7]
```diff
@@ -1,234 +1,99 @@
...
-Yuzu
+*For the news post, see: [The new osu!taiko mascot is here!](/home/news/2017-05-25-the-new-osutaiko-mascot-is-here).*
 
-#### Aliases
-Yuzu
+![Mocha](img/Mocha.png "Mocha")
 
-#### Birthday
-10/04/2000
+Mocha is the osu!taiko mascot. She was initially designed during the [sixth fanart contest](/community/contests/2) by [Crowie](/users/6894067), which had been ranked 21st from the polls.
```
Comment (TPGPL) [inline wiki/Mascots/en.md]: has been ranked or was ranked

File: wiki/Mascots/en.md [hunk 8]
```diff
@@ -1,234 +1,99 @@
...
-* ztrot (Catcher art, Concept design)
+![Ryuuta](img/Ryuuta.png "Ryuuta")
 
-#### Gender
-Male
+> He is the hot-blooded leader of the cheer squad. He has a good soul and helps those around him who are in distress by standing up and fighting for them!
 
-#### Date Joined
-22/06/2014
+一本木龍太 (Ryūta Ippongi) was the osu!catch chibi-fruit catcher that had joined in 2008, but was replaced by [Yuzu](#yuzu) in 2014. He was created by [iNiS Corporation](https://en.wikipedia.org/wiki/INiS) and was once part of the old website. He also had an appearance in [Yandere Simulator](https://yanderesimulator.com) as an NPC.
```
Comment (TPGPL) [inline wiki/Mascots/en.md]: appeared

File: wiki/Mascots/en.md [hunk 9]
```diff
@@ -1,234 +1,99 @@
...
-?
+![Don](img/Don.png "Don")
 
-#### Appearance
-Mania
+> Don is the protagonist of the [Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin) series. He is a taiko drum with a white edge and four legs, a red face (which appears to be moly orange), and a light blue body. Don's dream is to share the beauty of Taiko with the world. Three years have passed since he moved into the Wada House, and he's become quite popular around town. He has a monstrous appetite and sometimes has expensive shopping at the Wada House which may take over for the worst. He usually ends his sentences with "Ta-don", meaning "Ba-dum" in Japanese.
 
-#### Creator
-* Daru (Art)
+和田どん (Wada Don), also known as Don or Don-chan, was one of the osu! mascots for osu!taiko that joined in 2008-05. He is 48cm tall and weighs 100+ kilograms. He appears in the skin for osu!taiko. His art was designed by Yukiko Yokoo (横尾有希子) and was voiced by Narahashi Miki (楢橋 美紀).
```
Comment (TPGPL) [inline wiki/Mascots/en.md]: weighs more than 100 kilograms

File: wiki/Mascots/en.md (hunk 4 above)
Comment (MegaApplePi) [reply wiki/Mascots/en.md]: > with a lowercase `P`

going with `with a lowercase "p"`

> link to Sarumaru's profile?

I'm not sure what his user id is.

File: wiki/Mascots/en.md (hunk 5 above)
Comment (MegaApplePi) [reply wiki/Mascots/en.md]: just ignore for now, I missed Yuzu's and a few other news posts when I did #1001

File: wiki/Mascots/en.md (hunk 4 above)
Comment (TPGPL) [reply wiki/Mascots/en.md]: isnt it incorrect with ASG then?

**Pull request merged**